TLM/TLM/TrafficManagerMod.cs

```diff
@@ -11,37 +11,48 @@
 namespace TrafficManager {
 	public class TrafficManagerMod : IUserMod {
 
-		public static readonly string Version = "10.20";
+        public static readonly uint GameVersion = 184673552u;
+        public static readonly uint GameVersionA = 1u;
+        public static readonly uint GameVersionB = 12u;
+        public static readonly uint GameVersionC = 0u;
+        public static readonly uint GameVersionBuild = 5u;
 
-		public static readonly uint GameVersion = 184673552u;
-		public static readonly uint GameVersionA = 1u;
-		public static readonly uint GameVersionB = 12u;
-		public static readonly uint GameVersionC = 0u;
-		public static readonly uint GameVersionBuild = 5u;
+        // Note: `Version` is also used in UI/MainMenu/VersionLabel.cs
+        public static readonly string Version = "10.20";
 
-		public string Name => "TM:PE " + Version;
+#if LABS
+        public string Branch => "LABS";
+#elif DEBUG
+        public string Branch => "DEBUG";
+#else
+        public string Branch => "STABLE";
```

Comment:
I don't think we need to specify "STABLE" in the stable edition; it's not called "Minecraft STABLE" or "Discord STABLE", however it is called "Minecraft BETA" or "Discord CANARY" and such. Yes, it is technically the 'stable' version, but there's no need to specify

Reply:
We call them LABS and STABLE everywhere else so why not in the UI?

Reply:
Because it's used everywhere in the _development_ area, not necessarily where the users see it. The workshop page is not "Traffic Manager: President Edition STABLE", is it? For development it's important to specify the two, but for the end user it only confuses them.

Reply:
I think it's still useful to clearly differentiate the mod flavours in content manager, and I don't think users will get confused by the mod-name being the same as what the workshop shows in Steam. If we get users complaining about it, it's trivial to remove in a future release.

Reply:
Fine, but just let it be known I'm against this